docs/governance.rst

```diff
@@ -0,0 +1,129 @@
+time.
+
+The authoritative list of members and their roles (alphabetical sorting):
+* Tim Head (red)
+* Min Ragan-Kelley (team leader)
+* Yuvi Panda (red)
+* Erik Sundell (blue)
+* Carol Willing (red)
+* Your Name (red or blue)
+
+Team privileges
+~~~~~~~~~~~~~~~
+
+-  recognition
+-  team members can speak for the project in public
+-  merge privileges on the project repositories
+-  red members can nominate other individuals to team membership
+
+Team responsibilities
+~~~~~~~~~~~~~~~~~~~~~
+
+Each member of the Binder Project team is expected to spend a
+"reasonable" (intentionally vague) amount of time contributing to the
+technology used by Project Binder as well as maintaining
+``mybinder.org``. This might mean contributing features, making patches,
+responding to issues, reviewing PRs, teaching about BinderHub, making
+organizational connections, welcoming new contributors, or evangelizing
+and helping others deploy BinderHub. They should be receptive to
+comments and ideas on the GitHub issues or in the Binder gitter channel.
+They should strive to make project decisions that reflect these
+community opinions.
+
+This list is non-complete.
+
+Team roles
+~~~~~~~~~~
+
+-  Team leader. This is a person who has impasse-breaking authority to
+   make decisions if the team can not agree. This authority should
+   be used exceedingly sparingly. The team leader is expected to grow
+   team members so that they can pass on their position to them.
+-  Red member. You spend a large part of your time coordinating and
+   promoting the Binder Project as well as the service at
+   ``mybinder.org``. You are expected to coordinate your actions with
+   other red members so that the project presents a coherent position
+   towards the outside. You are expected to prepare and attend the
+   monthly team calls. Being a red member is not a life sentence. We
+   expect red members to teach and grow other members so that they can
+   pass their organizational responsibilities on to them. This encourages
+   is to increase diversity, reduce burn-out and allow team members to
+   shift their focus in reaction to changing life circumstances.
+-  Blue Member. This is where the journey of a new team member starts.
+   You spend a large
+   part of your time facilitating contributions from others to the
+   project (e.g. reviewing work, answering questions, maintaining
+   project infrastructure). We expect blue members to teach and grow
+   contributors that can become blue members. This encourages diversity
+   of the team. After building a good understanding of the technology
+   and social structure of the project you can change to a red member by
+   volunteering for tasks which a red member is expected to perform.
+   After a while of doing so the team will ask if you want to become a
+   red member and, if you agree, recognise you as such at a monthly team
+   meeting.
+
+There are no other specific roles, but we may revisit this in the
```

Comment:
What about one more status such as "inactive team member" that would be a place for people who want to take a break, move on to other things etc, and would signal that they're still "members" of the community, just not actively working on it now (and make a note that anyone is welcome to become active again if they are willing to take on the responsibilities once more)

Reply:
I really like the JOSS strategy for naming editors who have moved on: ["editors emeritus"](https://joss.theoj.org/about#editorial_board). It implies that they are members of the community, people you can potentially reach out to for general questions or mentorship (though not required), but also indicates that they are no longer involved in the day-to-day work of the project

Reply:
Do you feel it makes people nervous to become "emeritus" because it is seen as a one way street?

Reply:
That is a fair point @betatim; "emeritus" does imply a one-way street, and might not be appropriate to use instead of the "inactive" status. 

Would there be any reason to consider having both? If someone has moved-on and doesn't intend to come back any time in the near future but is willing to mentor it could be fitting? (However, adding another designation might just complicate matters?)

Reply:
Bunch of words I could think of that are related, maybe one works? retired, inactive, emeritus, paused, graduates, alumni, moved on, green (just another colour)

What do you think of the following role description:
"""
* xxx member, your life situation has changed so that you prefer to (temporarily) not take on the rights and responsibilities of being an active member. You can return to active membership by resuming your community activities. Your return will be announced at the next monthly team meeting.
"""

Reply:
That looks good! And I like the idea of sticking with colours - there are fewer connotations / previous associations than any of the other terms

Reply:
Done.

Red + blue + green = white! We need all the roles to create a bright light :)💡